dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -92,11 +92,23 @@ ifneq (,$(filter auto auto-ssh,$(IOTLAB_NODE)))
     $(error Could not find 'archi' for $(BOARD), update mapping in $(lastword $(MAKEFILE_LIST)))
   endif
 
-  _NODES_DEPLOYED = $(shell iotlab-experiment --jmespath='deploymentresults."0"' --format='" ".join' get $(_IOTLAB_EXP_ID) --print)
+  # There are several deprecated and incompatible features used here that were
+  # introduced between versions 2 and 3 of the IoT-LAB cli tools.
+  # For backward compatibility, we manage these changes here.
+  _CLI_TOOLS_MAJOR_VERSION ?= $(shell iotlab-experiment --version | cut -f1 -d.)
```

Comment:
Can you put all of this in a common file?

Reply:
If you provide a follow up PR I can ACK this one since it already passed murdock.